recipe/build.sh

```diff
@@ -1,24 +1,42 @@
 #!/bin/bash
 
-export LDFLAGS="$LDFLAGS $(pkg-config --libs ncurses)"
-export CPPFLAGS="$CPPFLAGS $(pkg-config --cflags-only-I ncurses)"
-export CFLAGS="$CFLAGS $(pkg-config --cflags-only-I ncurses)"
+# Prevent running ldconfig when cross-compiling.
+if [[ "${BUILD}" != "${HOST}" ]]; then
+  echo "#!/usr/bin/env bash" > ldconfig
+  chmod +x ldconfig
+  export PATH=${PWD}:$PATH
+fi
+
+
+export CPPFLAGS="${CPPFLAGS} -DSQLITE_ENABLE_COLUMN_METADATA=1 \
+                             -DSQLITE_ENABLE_UNLOCK_NOTIFY \
+                             -DSQLITE_ENABLE_DBSTAT_VTAB=1 \
+                             -DSQLITE_ENABLE_FTS3_TOKENIZER=1 \
+                             -DSQLITE_SECURE_DELETE \
+                             -DSQLITE_MAX_VARIABLE_NUMBER=250000 \
+                             -DSQLITE_MAX_EXPR_DEPTH=10000 \
+                             -DSQLITE_ENABLE_GEOPOLY \
+                             -DSQLITE_ENABLE_JSON1 \
+                             -DSQLITE_ENABLE_RTREE=1"
+
 
 if [ $(uname -m) == ppc64le ]; then
-    export B="--build=ppc64le-linux"
+    export PPC64LE="--build=ppc64le-linux"
 fi
 
-./configure SQLITE_ENABLE_RTREE=1 \
-            $B --enable-threadsafe \
-            --enable-json1 \
-            --enable-tempstore \
+./configure --prefix=${PREFIX} \
+            --build=${BUILD} \
+            --host=${HOST} \
+            --enable-threadsafe \
             --enable-shared=yes \
             --enable-readline \
             --disable-editline \
```

Comment:
@msarahan and @mingwandroid I noticed that in AR version of this recipe you have:

```
            --enable-readline    \
            --enable-editline    \
            --disable-readline   \
```

and `libedit` is used in the recipe.

I wonder what is the reason and if we should follow it here or not.
Ping @conda-forge/core

Reply:
That part I know :smile: 

I wonder if we should do it in `conda-forge` too. I hear bad things about `libedit` but I never really used it. If that is "good enough" I guess we could switch.

Reply:
Personally, I don't use the `sqlite` REPL so I have no need for `readline` or `libedit` support.  Hearing the opinions of folks who do make use of the REPL would be helpful in this decision.  

For Python, I am in favor of keeping `readline`. The Python REPL with libedit is less functional that when using `libedit`.

Reply:
@mingwandroid I guess that the first `--enable-readline` there is unnecessary, right? All we need to try `libedit` would be:

```
--enable-editline    \
--disable-readline   \
```

correct?

Reply:
> For Python, I am in favor of keeping `readline`. The Python REPL with `libedit` is less functional that when using `libedit`.

That is what concerns me when considering moving to `libedit`. You are not the only one I know that complains about it and prefers `readline`. So let's not touch that for now :grimacing:

Regarding the licenses, if one interprets the metadata in the recipe are the Software license, then we are good, but if that is interpreted as the final **package** license, then we probably say GPL instead. But that is a discussion for later :smile: